File: API/API 12.5 oficial/UnimindAPI/Competicao.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using static System.Runtime.InteropServices.JavaScript.JSType;

[Table("Competicao", Schema = "unimind")]
public class Competicao
{
    [Key]
    [Column("idCompeticao")]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
    public int IdCompeticao { get; set; }

    [Column("data")]
    public DateOnly Data { get; set; }

    [Column("idUsuario1")]
    public int IdUsuario1 { get; set; }

    [Column("idUsuario2")]
    public int IdUsuario2 { get; set; }

    [Column("idNivel")]
    public int IdNivel { get; set; }
}

File: app/src/main/java/com/example/unimind/data/Competicao.kt
package com.example.unimind.data
import com.google.gson.annotations.SerializedName

data class Competicao(
    @SerializedName("idCompeticao") val idCompeticao: Int,
    @SerializedName("data") val data: String,
    @SerializedName("idUsuario1") val idUsuario1: Int,
    @SerializedName("idUsuario2") val idUsuario2: Int,
    @SerializedName("idNivel") val idNivel: Int
)
